File: API/Models/ProductCategory.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace surstroem.Models
{
    public partial class ProductCategory
    {
        public int Id { get; set; }
        public int? ProductId { get; set; }
        public int? CategoryId { get; set; }
        public DateTime? CreatedAt { get; set; }
        public DateTime? UpdatedAt { get; set; }

        public virtual Category Product { get; set; }
    }
}

File: php/app/Models/product_category.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Model as Eloquent;
class product_category extends Eloquent
{
    protected $fillable = [
        'product_id', 'category_id'
    ];

    public function product() {
        return $this->hasOne(product::class, 'id', 'product_id');
    }

    public function category() {
        return $this->hasOne(category::class, 'id', 'category_id');
    }
}
